src/datasources/db/userTickets.ts

```diff
@@ -17,18 +25,30 @@ export const userTicketsApprovalStatusEnum = [
 
 export const userTicketsRedemptionStatusEnum = ["redeemed", "pending"] as const;
 
+export enum UserTicketGiftStatus {
+  Pending = "pending",
+  Accepted = "accepted",
+  Rejected = "rejected",
+  Cancelled = "cancelled",
+  Expired = "expired",
+}
+
 // USER-TICKETS-TABLE
 export const userTicketsSchema = pgTable(
   "user_tickets",
   {
     id: uuid("id").primaryKey().notNull().defaultRandom(),
     publicId: uuid("public_id").notNull().defaultRandom(),
-    userId: uuid("user_id").references(() => usersSchema.id),
+    userId: uuid("user_id")
+      .references(() => usersSchema.id)
+      .notNull(),
     ticketTemplateId: uuid("ticket_template_id")
       .references(() => ticketsSchema.id)
       .notNull(),
     purchaseOrderId: uuid("purchase_order_id")
-      .references(() => purchaseOrdersSchema.id)
+      .references(() => purchaseOrdersSchema.id, {
```

Comment:
En ocasiones necesitaba borrar compras desde el drizzle-studio y era tedioso tener que borrar todos los tickets primero.

Reply:
![image](https://github.com/user-attachments/assets/502a6ab1-0c27-4de5-b947-40098fd03106)

Reply:
got you 👍